javatests/arcs/core/storage/CapabilitiesResolverTest.kt

```diff
@@ -128,21 +128,28 @@ class CapabilitiesResolverTest {
             resolver1.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId),
             VolatileStorageKey::class.java
         )
-        verifyStorageKey(
-            resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId),
-            RamDiskStorageKey::class.java
-        )
-        verifyStorageKey(
-            resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId),
-            DatabaseStorageKey::class.java
-        )
+        val ramdiskKey =
+            resolver1.createStorageKey(Capabilities.TiedToRuntime, thingSchema, handleId)
+        verifyStorageKey(ramdiskKey, RamDiskStorageKey::class.java)
+        assertThat(ramdiskKey).isEqualTo(resolver1.createStorageKey(
+            Capabilities.TiedToRuntime,
+            thingSchema,
+            handleId
+        ))
+
+        val persistentKey =
+            resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId)
+        verifyStorageKey(persistentKey, DatabaseStorageKey::class.java)
+        assertThat(persistentKey)
+            .isEqualTo(resolver1.createStorageKey(Capabilities.Persistent, thingSchema, handleId))
```

Comment:
Same here?

Reply:
done. 
kotlin is a language optimized for people with veeeeery narrow screens!